src/geos.h

```diff
@@ -111,6 +112,8 @@ enum {
       PyErr_WarnFormat(PyExc_Warning, 0,                                                 \
                        "Invalid GeoJSON: geometry is returned as None. %s", last_error); \
       break;                                                                             \
+    case PGERR_PYSIGNAL:                                                                \
```

Comment:
Suggest adding a comment that this error is raised on an interrupt signal does not raise its own exception or warning.